docs/documentation/getting-started.md

```diff
@@ -324,7 +121,28 @@ You can then delete the first account, add a new account with the right values,
 Once the other user has accepted your contact request, they will request you adding them as a contact as well.
 This will automatically be displayed by the plugin as a dialog when you are connected to the account.
 
-### Starting a Session - Host
+{% endcapture %}
+{% include ide-tabs.html eclipse=eclipse intellij=intellij %}
+
+### Step 4: Starting and Joining Sessions
+
+You can work together with your contacts by either **starting your own
+session** or by **being invited to one**.
+
+When **starting** a session, **Saros will copy the project** to the
+computer of your invited contacts.
```

Comment:
Simplification as we will not always "copy" the project to the other computer. Our goal is to create the same state on both sides, so we only transfer the needed resources.